sdcm/nemesis.py

```diff
@@ -584,12 +584,13 @@ def disrupt_truncate(self):
         # do the actual truncation
         self.target_node.run_cqlsh('TRUNCATE {}.{}'.format(keyspace_truncate, table))
 
-    def _modify_table_property(self, name, val):
+    def _modify_table_property(self, name, val, filter_out_table_with_counter=False):
         disruption_name = "".join([p.strip().capitalize() for p in name.split("_")])
         self._set_current_disruption('ModifyTableProperties%s %s' % (disruption_name, self.target_node))
 
         ks_cfs = self.loaders.get_non_system_ks_cf_list(loader_node=random.choice(self.loaders.nodes),
-                                                        db_node=self.target_node)
+                                                        db_node=self.target_node,
+                                                        filter_out_table_with_counter=filter_out_table_with_counter)
         keyspace_table = ks_cfs[0] if ks_cfs else ks_cfs
         if not keyspace_table:
             self.log.error('Non-system keyspace and table are not found. ModifyTableProperties nemesis can\'t be run')
```

Comment:
Better to throw exception so we can identify the error in events

Reply:
Fixed